CONTRIBUTING.md

```diff
@@ -218,7 +213,18 @@ It can be unassigned or moved to any other project that is provided.
 In every project, an issue passes four stages: `Backlog`, `In progress`, `Review in progress`, and `Done`,
 independent of their association to a specific milestone.
 
-### Contact Us
+### Releases
+
+Please find more information about our release approach [here](docs/developer/releases.md).
+
+## Contact Us
 
 If you have questions or suggestions, do not hesitate to contact the project developers via
 the [project's "dev" list](https://dev.eclipse.org/mailman/listinfo/dataspaceconnector-dev). 
+
+You may also want to join our [Discord server](https://discord.gg/n4sD9qtjMQ).
+
+There, we provide a biweekly meeting on fridays 2-3 p.m. CET to give any interested person the
```

Comment:
Maybe a link to the discussion would be helpful here to know in which weeks the meeting takes place.

Reply:
Done